.github/workflows/release-pr-rust.yml

```diff
@@ -81,6 +81,23 @@ jobs:
           cargo release version --execute --no-confirm \
             -p "${{ inputs.crate }}" "${{ inputs.version }}"
 
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2.4.0
+        with:
+          version: 8
+          run_install: false
+
+      - name: Setup and build
+        uses: ./.github/actions/setup-and-build
+
+      - name: Commit IDL and other changes
```

Comment:
I think you could remove this step - the IDL changes generated in the previous step would end up in the commit we create at the end.

Also, I think that with the current approach, the "Include IDL..." commit would include the Rust changes as well, because we run `cargo release version`before? And then the last step would fail, because of no files left to commit?

Reply:
i'm testing it out right now. the goal of this pr is to commit the idl changes caused by the rust version bumps